simple_kafka_client/SimpleKafkaClient/Consumer.php

```diff
@@ -0,0 +1,90 @@
+<?php
+declare(strict_types=1);
+
+namespace SimpleKafkaClient;
+
+use SimpleKafkaClient;
+
+class Consumer extends SimpleKafkaClient
+{
+    /**
+     * Consumer constructor.
+     * @param Configuration $configuration
+     * @throws Exception
+     */
+    public function __construct(Configuration $configuration) {}
+
+    /**
+     * @param TopicPartition[]|null $topicPartitions
+     * @throws Exception
+     */
+    public function assign(?array $topicPartitions): void {}
+
+    /**
+     * @return array
+     * @throws Exception
+     */
+    public function getAssignment(): array {}
+
+    /**
+     * @param string[] $topics
```

Comment:
Could you also do the changes on each array doc blocks please?
It is hard to know what is in the output without knowing more about this extension ^^

Reply:
I totally agree, sry i hope i got them all now :v: